databases/week3/meal_sharing.sql

```diff
@@ -0,0 +1,196 @@
+WHERE ID = 1;
+
+-- Review
+-- Queries to write
+-- Get all reviews
+SELECT *
+FROM REVIEW;
+
+-- Add a new review
+INSERT INTO review(title, `description`, meal_id, stars, created_date)
+VALUES("Most boring food ever!", "I tried boring food in my life but this is by far the most boring ever!!",
+2, 1, now());
+
+-- Get a review with any id, fx 1
+SELECT * 
+FROM review
+WHERE id = 2;
+
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE review
+SET stars = 0
+WHERE id = 1;
+
+-- Delete a review with any id, fx 1
+DELETE from review
+WHERE id = 1;
+
+
+
+-- ADDITIONAL QUERIES
+-- add new meals
+INSERT INTO meal (title, `description`, location, `when`, max_reservations, price, created_date)
+VALUES ("Continental breakfast", "Best breakfast you'll ever try",
+ "Aarhus, Denmark", "2021-04-25 10:00:00", 100, 15.00, NOW());
+ INSERT INTO meal (title, `description`, location, `when`, max_reservations, price, created_date)
+VALUES ("Complete dinner buffet", "All you can eat!",
+ "Copenhagen, Denmark", "2021-05-02 10:00:00", 80, 60.00, NOW());
+ INSERT INTO meal (title, `description`, location, `when`, max_reservations, price, created_date)
+VALUES ("Workers day brunch", "Start your day with a world-class brunch!",
+ "Helsingor, Denmark", "2021-05-01 10:00:00", 90, 30.00, NOW());
+ INSERT INTO meal (title, `description`, location, `when`, max_reservations, price, created_date)
+VALUES ("All you can eat BBQ", "From the best grill in town",
+"Copenhagen, Denmark", "2021-04-28 10:00:00", 60, 15.00, NOW());
+
+-- add new reservations
+INSERT INTO reservation( number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(4, 2, now(), "111-500-1312", "Jesus", "jesus@gmail.com");
+INSERT INTO reservation( number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(6, 5, now(), "0000-1111", "Martin Martin", "mmartin@gmail.com");
+INSERT INTO reservation( number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email)
+VALUES(2, 5, now(), "98999091", "Carlos Eltopo Kejira", "cbandi@gmail.com");
+
+-- add new reviews
+INSERT INTO review(title, `description`, meal_id, stars, created_date)
+VALUES("Absolute trash!", "I couldnt finish my food",
+2, 1, now());
+INSERT INTO review(title, `description`, meal_id, stars, created_date)
+VALUES("Best bbq ever!", "Amazing food and all you can eat! Definitly coming back",
+6, 5, now());
+INSERT INTO review(title, `description`, meal_id, stars, created_date)
+VALUES("OK breakfast", "Nothing out of ordinary",
+3, 3, now());
+
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * 
+from meal
+WHERE PRICE < 50;
+
+-- Get meals that still has available reservations
+SELECT title, max_reservations, SUM(number_of_guests) AS reserved
+FROM meal
+JOIN reservation
+ON meal.id = meal_id
+GROUP BY meal_id
+HAVING max_reservations > SUM(number_of_guests);
```

Comment:
This query doesn't return the meals with no reservations at all. What do you need to change to achieve that?